test/features/step_definitions/signInSteps.js

```diff
@@ -0,0 +1,52 @@
+// Step definitions
```

Comment:
code style needs to align with standardJS also use const/let instead of var accordingly